bobber/lib/analysis/table.py

```diff
@@ -146,6 +148,38 @@ def fio_iops(results: list) -> Tuple[list, list]:
         return [read, write]
 
 
+def fio_125k_bw(results: list) -> Tuple[list, list]:
+    """
+    Save the FIO 125k bandwidth read and write results.
+
+    Save the read and write results from the FIO 125k bandwidth tests on an
+    increasing per-system basis with the first element in the list being the
+    column header.
+
+    Parameters
+    ----------
+    results : list
+        A ``list`` of ``dictionaries`` containing all results from the tests.
+
+    Returns
+    -------
+    tuple
+        Returns a ``tuple`` of (``list``, ``list``) containing the read and
+        write 125k bandwidth results, respectively.
+    """
+    try:
+        read = [FIO_125K_READ_BW] + [bytes_to_gb(result[1]['125k_bandwidth']
```

Comment:
What does the `1` in `result[1]` refer to? Is that the system count? If so, I wonder if this will capture results for multi-node tests, but I know we can't test multi-node right this second. If I remember correctly, the structure I had for the dictionaries was that each key denoted the number of systems being tested.